### PR TITLE
Adding update risks for OCPBUGS-27261

### PR DESCRIPTION
**Pull request by LalatenduMohanty**

### CODE REVIEW

File: blocked-edges/4.14.0-NodesOnAWSWithCustomDomainNameWillBeNotReady.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+to: 4.14.0
+from: 4[.]13[.].*
+
+url: https://issues.redhat.com/browse/MCO-1031
+name: NodesOnAWSWithCustomDomainNameWillBeNotReady
```
Comment (petr-muller) [inline blocked-edges/4.14.0-NodesOnAWSWithCustomDomainNameWillBeNotReady.yaml:5]: `AWSCustomDomainNodesNotReady` perhaps?

File: blocked-edges/4.14.0-NodesOnAWSWithCustomDomainNameWillBeNotReady.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+message: |-
+  Nodes on AWS with custom domain names will go to NotReady state.
+matchingRules:
+- type: PromQL
+  promql:
+    promql: |
+      (
+        group by (type) (cluster_infrastructure_provider{type="AWS"})
```
Comment (petr-muller) [inline blocked-edges/4.14.0-NodesOnAWSWithCustomDomainNameWillBeNotReady.yaml:13]: We should add the `__id=""` placeholders for hypershi

**Review by petr-muller — APPROVED** [review]
/hold

LGTM, left comments proposing some non-critical changes, feel free to just unhold and merge, or address if you feel like it

File: blocked-edges/4.14.0-NodesOnAWSWithCustomDomainNameWillBeNotReady.yaml (hunk 1 above)
Comment (LalatenduMohanty) [reply blocked-edges/4.14.0-NodesOnAWSWithCustomDomainNameWillBeNotReady.yaml:5]: sounds good going pivot to this

**Review by PratikMahajan — APPROVED** [review]
/lgtm

**Pull request merged**